From 6f66f7bafb52e9f6b0023ebfe9c89c4ad34a7a33 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 14 May 2021 15:05:11 +0000
Subject: [PATCH] libstdc++: fix incomplete type of StartPresentationContext

std::move of |context_| requires full definition of
media_router::StartPresentationContext.
---
 .../presentation_request_notification_item.h                 | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/chrome/browser/ui/global_media_controls/presentation_request_notification_item.h b/chrome/browser/ui/global_media_controls/presentation_request_notification_item.h
index 931fdb2..b410fee 100644
--- a/chrome/browser/ui/global_media_controls/presentation_request_notification_item.h
+++ b/chrome/browser/ui/global_media_controls/presentation_request_notification_item.h
@@ -11,14 +11,11 @@
 #include "base/memory/weak_ptr.h"
 #include "chrome/browser/ui/global_media_controls/media_notification_service_observer.h"
 #include "components/media_message_center/media_notification_item.h"
+#include "components/media_router/browser/presentation/start_presentation_context.h"
 #include "content/public/browser/presentation_request.h"
 
 class MediaNotificationService;
 
-namespace media_router {
-class StartPresentationContext;
-}  // namespace media_router
-
 class PresentationRequestNotificationItem final
     : public media_message_center::MediaNotificationItem {
  public:
-- 
2.26.3

